source/MaterialXGraphEditor/RenderView.cpp

```diff
@@ -169,21 +175,23 @@ RenderView::RenderView(const std::string& materialFilename,
     // current working directory into account.
     mx::FileSearchPath localSearchPath = searchPath;
     localSearchPath.append(mx::FilePath::getCurrentPath());
-    _materialFilename = localSearchPath.find(_materialFilename);
+    //_materialFilename = localSearchPath.find(_materialFilename);
     _meshFilename = localSearchPath.find(_meshFilename);
     _envRadianceFilename = localSearchPath.find(_envRadianceFilename);
 
     // Set default Glsl generator options.
     _genContext.getOptions().targetColorSpaceOverride = "lin_rec709";
     _genContext.getOptions().fileTextureVerticalFlip = true;
     _genContext.getOptions().hwShadowMap = true;
+    // Make sure all uniforms are added so value updates can
+    // find the the corresponding uniform.
+    _genContext.getOptions().shaderInterfaceType = mx::SHADER_INTERFACE_COMPLETE;
```

Comment:
Some uniforms in node graphs were not being exposed so material value update would fail.
Make sure all ShaderGraph uniforms are exposed.